Documents-Endpoint.md

```diff
@@ -249,19 +364,491 @@ Retrieve the full document bgu;11;2029
+
+### PUT Query parameters
+
+The three required parameters for a `PUT` Document request are `id`, `token` (if supported by the implementation), and `ref`. Only one structural segment of the document may be modified in a single `PUT` request.
+
+The `ref` reference determines the structural depth of the modification. Suppose a document has a three-level citation structure represented in XML by nested `<div>` elements. A `PUT` request with a `ref` of "3.16.8" will replace just one `<div>` element at the lowest structural level. If the request is instead sent with a `ref` parameter of "3.16", the submitted `<div>` element will replace the second-level `<div>` with the identifier "3.16". In the latter case, the submitted `<div>` element will need to contain the same series of nested `<div>` elements that represent the existing bottom-level segments in the server's version of the file: "3.16.1", "3.16.2", "3.16.3", etc. It is recommended that implementations check the submitted text before performing upper-level modifications like this, to ensure that no lower-level structural segments are being created or destroyed. Alternately, a client may avoid this issue by only making `PUT` requests at the lowest level of the document's citation structure.
+
+### PUT Request Body
+
+The body of the `PUT` request must be wrapped in a TEI rootnode containing a `<dts:fragment>` entity. See the further specifications under "Default Request and Response Body Format" above.
+
+The contents of this `<dts:fragment>` must be a single XML element (along with its enclosed text and/or children) representing the modified form of the target document section. This single element inside the `<dts:fragment>` will replace the corresponding XML element in the document on the server.   
+
+The text/XML submitted in the body of a Document's `PUT` request will completely replace the XML entity representing the identified text segment. So the submitted fragment __must include the outer element__ identified by the "ref" value. If you identify your submitted text as a modified version of line "12", and if each line is represented by a TEI `<ln>` element, you would include the opening and closing tags `<ln xml:id="16">` and `</ln>` around the changed content. This is to allow modification of the attributes on the outer element tag.
```

Comment:
should the example `<ln xml:id="16">` be `<ln xml:id="12">` ?

Reply:
And there's no `<ln>` element in TEI. `<l>`?

Reply:
Fixed. Thanks.